docs/gnmi_extensions.md

```diff
@@ -0,0 +1,76 @@
+# gNMI extensions on the Northbound interface
+
+gNMI is designed primarily as a device management interface, and augmenting it
+with the extra functionality to do network wide management of the configuration
```

Comment:
minor: I would change to "to do network wide configuration management across..."

Reply:
done